packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/FIAPaymentQueueHandler.m

```diff
@@ -233,8 +233,90 @@ - (BOOL)paymentQueue:(SKPaymentQueue *)queue
   return self.queue.transactions;
 }
 
-- (SKStorefront *)storefront {
+- (SKStorefront *)storefront API_AVAILABLE(ios(13.0)) {
   return self.queue.storefront;
 }
 
+@synthesize delegate;
```

Comment:
why can't use auto synthesize

Reply:
i think property on protocols cant be auto synthesized?
could be wrong tho
https://stackoverflow.com/questions/16355599/auto-property-synthesis-will-not-synthesize-property-declared-in-protocol-whi